applications/nrf_desktop/src/modules/ble_conn_params.c

```diff
@@ -240,14 +305,12 @@ static bool app_event_handler(const struct app_event_header *aeh)
 		const struct module_state_event *event =
 			cast_module_state_event(aeh);
 
-		if (check_state(event, MODULE_ID(ble_state),
-				MODULE_STATE_READY)) {
+		if (check_state(event, MODULE_ID(ble_state), MODULE_STATE_READY)) {
```

Comment:
the last change is fine.

I understand the problem is happening because usb_state event comes before everything is initialized but if this is the case should we block the reschedule or work will take care of recognizing the fact that connections are not made yet?